### PR TITLE
Add config file for stale bot

### PR DESCRIPTION
**Pull request by nikkixdev**
Signed-off-by: Nikki Attea <nikki@sensu.io>

## What is this change?

Adds config file for a github bot https://probot.github.io/apps/stale/

## Why is this change necessary?

https://trello.com/c/rTfspzBD/41-start-utilizing-github-bots-https-probotgithubio-apps

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```
Comment (echlebek) [inline .github/stale.yml:8]: can you add "bug" to this?

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
```
Comment (echlebek) [inline .github/stale.yml:2]: I think this needs to be larger, perhaps 180 or so?

Comment (nikkixdev) [reply .github/stale.yml:2]: We could increase it to 180. The bot comments at `daysUntilStale` and then waits `daysUntilClose` until actually closing if no response is received.

**Comment by nikkixdev** [comment]
See also https://github.com/sensu/sensu-enterprise-go/pull/1023

**Pull request merged**